product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx

```diff
@@ -5,14 +5,17 @@ legacyRedirectsGenerated:
   - "/edb-docs/d/edb-postgres-enterprise-manager/installation-getting-started/linux-installation-guide/8.0/installing_pem_server_and_apache_web_server_preferences.html"
 ---
 
-During the PEM server installation, you can specify your hosting preferences for the Apache Web Server(PEM-HTTPD):
+During the PEM server installation, you can specify your hosting preferences for the Apache Web Server:
 
-**To install the PEM Server and Apache Web Server (PEM-HTTPD) on the same host**
+## To install the PEM Server and Apache Web Server on the same host
 
-Follow the installation steps; while running the configuration script, select the `Web Services and Database` option to install PEM Server and Apache Web Server on the same host.
+1.  Install the PEM Server, for steps see [Installing the PEM Server on Linux](03_installing_pem_server_on_linux).
```

Comment:
The link to Installing the PEM Server on Linux topic doesn't work in both sections

Reply:
Fixed the all the broken links and pushed the changes